mathOperations_hw/index.js

```diff
@@ -0,0 +1,12 @@
+let firstDigit = prompt(`Please enter the first digit.`);
+let secondDigit = prompt(`Please enter the second digit.`);
+
+firstDigit = Number(firstDigit);
```

Comment:
You can combine methods to get the desired result so it will ok :
let firstDigit = Number(prompt(`Please enter the first digit.`));